spark/src/main/kotlin/com/adevinta/spark/components/text/TextLink.kt

```diff
@@ -115,80 +106,80 @@ import kotlinx.coroutines.launch
  * text, baselines and other details. The callback can be used to add additional decoration or
  * functionality to the text. For example, to draw selection around the text.
  * @param style style configuration for the text such as color, font, line height etc.
- * @param onClick callback when textLink part is clicked.
+ * @param icon The optional icon to be displayed at the start or the end of the Text.
+ * @param iconColor [Color] to applied to the icon. If [Color.Unspecified], and [style] has no color set,
+ * this will be [LocalContentColor]. usually should be set to the color of the text not links
+ * @param iconSide If an icon is added, you can configure the side where is should be displayed, at the start or end of the button
+ * @param onClick callback when textLink container is clicked.
  */
 
+@SuppressLint("VisibleForTests")
 @Composable
 public fun TextLink(
-    textFull: String,
-    textLink: String,
+    text: AnnotatedString,
+    onClickLabel: String,
     modifier: Modifier = Modifier,
-    colorText: Color = Color.Unspecified,
-    colorLink: Color = colorText,
-    fontSize: TextUnit = TextUnit.Unspecified,
```

Comment:
We just need to re-add these arguments

Reply:
Provided a PR, can you kindly have a look
https://github.com/adevinta/spark-android/pull/897